src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1960,7 +1985,7 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
                windows_admin_password=None,
                enable_ahub=False,
                kubernetes_version='',
-               node_vm_size="Standard_DS2_v2",
+               node_vm_size=None,
```

Comment:
Will the modification of the default value lead to a breaking change?

Reply:
I could help answer this question. Changing the default value of the parameter to None is to determine whether the user has provided a value. After introducing snapshot, we may also read this parameter from a snapshot object. By priority, we take the value from user input > snapshot object > default value (`Standard_DS2_v2`).

Reply:
no, in get_node_vm_size in decorator.py, the node_vm_size still defaults to "Standard_DS2_v2" if it is not specified from customer, or not from a snapshot etc.